### ISSUE TITLE
TEST & RUN mode does not work.

### ISSUE THREAD

**Issue by Bamike1** [FlutterFlow/flutterflow-issues#1910]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I am unable to TEST or RUN my project though I have successfully downloaded the project APK multiple times. 

### Expected Behavior

I expect to be able to test the edits and Run the app to ensure that everything works perfectly and I am unable to do that making it difficult for me to proceed 

### Steps to Reproduce

Click Test or Run on the VAAGE project

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT48l8nfvJNgrscB0LrfbcFViikUJl87UIIRjetudxU8J5zKB+wAYvf7P1FWTL2lYW9IOmH9hnEy/PfqiPzLNO0DFzOsb6ZB1qVybDvMbDigRJvWDsyBfW1fM/tTJ2qe0qS3kRQlGNNZdXwSwF62DOnJcDbAQJjFNTIFROW1KdeK2SrDX1iXc2URm05KZDPz

### Context

I am trying to build an app that will be a bridge/connector between business owners and expert virtual support professionals. The app is expected to have 3 categories of users - The Expert, The Client, and the Student ) who want to become an expert and we have a page where they choose the category they belong to for future reference.

To this end, this app will have 5 protected pages that can only be accessed by users with paid subscriptions - 

* The client feed where a list of jobs/prospects can be seen by the experts
* The expert feed where a list of experts promoting their skills and services can be seen by the clients
* The job page - where clients write their job listings. The client feed gets it's information from this page
* The Expert page - where experts share information about their skills. The Expert feed gets its information from this page
* The course page where students can take courses

To properly set this up, there is a lot of data reading and transfer and my inability to test or run is preventing me from finding out if my build is correct and if I am ready for beta testers.

Please help

### Visual documentation

![Screen Shot 2023-11-26 at 11 41 55 PM](https://github.com/FlutterFlow/flutterflow-issues/assets/152103066/c67ad5ff-5f2b-450a-a2c9-37eaeaaaa707)

### Additional Info

None

### Environment

```markdown
- FlutterFlow version: v4.0
- Platform: web
- Browser name and version: Google Chrome
- Operating system and version affected: macOS
```

**Comment by hariprasadms** [comment]
Hi @Bamike1 - This is mostly project specific issue. Are you still experience the issue ?

**Comment by Bamike1** [comment]
No, I tried using a different laptop [ and I was able to test my project
